From ecaabab3b9e24ba94442dc2401e876592dc4c4e6 Mon Sep 17 00:00:00 2001
From: Thomas Preud'homme <thomas.preudhomme@celest.fr>
Date: Fri, 15 Jul 2011 18:13:56 +0200
Subject: Don't define strtold and strtof on Debian kFreeBSD

Don't define strtold and strtof on Debian GNU/kFreeBSD as they are
already defined there.

Origin: vendor
Forwarded: http://lists.nongnu.org/archive/html/tinycc-devel/2011-07/msg00025.html
Last-Updated: 2011-07-15
---
 tcc.h |    3 +--
 1 files changed, 1 insertions(+), 2 deletions(-)

diff --git a/tcc.h b/tcc.h
index d8b4400..1edf31b 100644
--- a/tcc.h
+++ b/tcc.h
@@ -819,8 +819,7 @@ enum tcc_token {
   #define strtoll (long long)strtol
 #endif
 #elif defined(TCC_UCLIBC) || defined(__FreeBSD__) \
-    || defined(__FreeBSD_kernel__) || defined(__DragonFly__) \
-    || defined(__OpenBSD__)
+    || defined(__DragonFly__) || defined(__OpenBSD__)
 /* currently incorrect */
 static inline long double strtold(const char *nptr, char **endptr)
 {
-- 
1.7.5.4

